README.md

```diff
@@ -52,10 +52,11 @@ prompt([options, parentBrowserWindow]).then(...).catch(...)
 | ------------- | ------------- |
 | title  | (optional, string) The title of the prompt window. Defaults to 'Prompt'. |
 | label  | (optional, string) The label which appears on the prompt for the input field. Defaults to 'Please input a value:'. |
+| buttonLabels | (optional, object) The text for the OK/cancel buttons. Properties are 'ok' and 'cancel' . Defaults to null. |
```

Comment:
Seems like there's one space too much here, otherwise the text is fine

Reply:
Sorry, missed that! About to commit that update..